### ISSUE TITLE
Ability to publish multiple packages

### ISSUE THREAD

**Issue by baseonmars** [rocjs/roc-repo#3]
To my eye there is no easy way to publish multiple packages - for example if one makes a change to the babel configuration. This kind of things is more likely to pop up early in a repositories life cycle but could appear later on.

I'd like to suggest a few ways of achieving this, I guess I'd ultimately like to see all of these implemented.

1. Add an `ALL` scope which triggers a status change for all packages. e.g. `feat(ALL): now uses transform-runtime`.
2. Allow multiple scopes, possibly after the first line as this is cramped as it is. e.g. 

```
feat(multi): fix usage of Foo in packages

scopes: roc-plugin-foo, roc-plugin-bar
```
3. All an interactive publishing mode e.g. `roc publish --interactive`, which would allow you to indicate which packages should be released and what their version should be.
4. A force mode, which allows you to force a number of packages to be published `roc publish roc-plugin-foo roc-plugin-bar --force --patch` - if no version bump is supplied either go interactive or throw an error requiring one of `--patch`, `--minor`, `--major`

**Comment by dlmr** [comment]
> 3. All an interactive publishing mode e.g. roc publish --interactive, which would allow you to indicate which packages should be released and what their version should be.

This is now the default behaviour, the previous automatic mode can be found under `roc release --automatic`.

**Comment by dlmr** [comment]
@baseonmars 1 and 2 have now been implemented and released as `roc-plugin-repo@0.0.19`.

I don't think we need a force mode (4), can you motivate why this is needed? The reason why I'm skeptical is that it would defeat the purpose of the CHANGELOG generation and that it's still possible to do releases manually.
